PlayCover/en.lproj/Localizable.strings

```diff
@@ -84,10 +84,10 @@
 
 "settings.tab.graphics" = "Graphics";
 "settings.picker.iosDevice" = "iOS device:";
-"settings.picker.adaptiveRes" = "Adaptive resolution:";
-"settings.picker.adaptiveRes.help" ="Adapts and scales app window to fit your display.";
-"settings.picker.adaptiveRes.0" = "Off";
-"settings.picker.adaptiveRes.1" = "Auto";
+"settings.picker.adaptiveRes" = "Resolution:";
+"settings.picker.adaptiveRes.help" ="Scales the app's window resolution";
```

Comment:
"Scale" implies resizing, not changing resolution

Reply:
"Changes the ..." is good?